File: examples/extending-objects/Interfaces/Kotlin/Employee.kt
class Employee(firstName : String, lastName : String, age : Int, staffNumber : Int) : Person {
  override firstName : String 
  override lastName : String
  var age : Int
  var staffNumber : Int
    
  override fun toString() : String {
    return this.firstName + " " + this.lastName + " " + this.age + ", " + this.staffNumber;
  }
}

File: examples/extending-objects/Interfaces/C#/Employee.cs
public class Employee : Person {
  private firstName, lastName;
  private int age, staffNumber;
  
  Employee(String firstName, String lastName, int age, int staffNumber) {
    this.firstName = firstName;
    this.lastName = lastName;
    this.age = age;
    this.staffNumber = staffNumber;
  }
  
  public String ToString() {
    return this.firstName + " " + this.lastName + ", " + age + ", " + this.staffNumber;
  }
}